index.html

```diff
@@ -15,15 +15,19 @@
         <select id="selectedEpisode"
           >Select an episode</select
         >
-        <form id="search-episodes">
-          <input type="text" placeholder="Search episodes..." />
+        <form>
+          <input
+            id="searchEpisode"
+            type="text"
+            placeholder="Search episodes..."
+          />
         </form>
         <span id="searchResult">Displaying 73/73 episodes</span>
```

Comment:
Here I think it is better to make it dynamic rather than putting static number, to make it suitable for any show,you can do that through javascript by putting allEpisodes.length instead of 73

Reply:
^ agreed. As soon as you move on to level 350, this static approach won't work for you